base/serialize.jl

```diff
@@ -116,16 +114,16 @@ function serialize(s::SerializationState, t::Tuple)
         writetag(s.io, LONGTUPLE_TAG)
         write(s.io, Int32(l))
     end
-    for i = 1:l
-        serialize(s, t[i])
+    for e in t
```

Comment:
`for elem in` appears only once in the code while `for x in` is used a lot. Once people get used to a convention, it can work even if it's not very explicit.

Anyway, this example is so short that either solution is perfectly fine IMHO.